src/screens/Dashboard/Dashboard.js

```diff
@@ -507,17 +545,15 @@ export default function HrDashboard(props) {
                           className="list-unstyled list mb-0"
                           style={{ height: `${notificationHeight}px` }}
                         >
-                          {allRequest &&
-                            allRequest.length > 0 &&
-                            allRequest.map((ele, index) => {
-                              const date = ele.created_at.split(" ")[0];
-                              const time = ele.created_at.split(" ")[1];
+                          {allRegularizationRequest &&
```

Comment:
This is extra ,
we have already used allRegularizationRequest.length > 0 and also we have used optionalchain so above line is not required.

Reply:
resolved